project/tests/com/google/daggerquery/server/ServerTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package com.google.daggerquery.server;
+
+import org.apache.http.client.utils.URIBuilder;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+public class ServerTest {
+
+  private static URIBuilder uriBuilder;
+
+  @BeforeClass
+  public static void startServer() {
+    // We can pass null since these arguments are not used.
+    Server.main(/*args =*/ null);
+
+    uriBuilder = new URIBuilder()
+      .setScheme("http")
+      .setHost("localhost")
+      .setPort(4921)
+      .setPath("/daggerquery/");
```

Comment:
Would it be possible to re-use the static final variables from the server class, maybe by making them public static final?

Reply:
Yep, [done](https://github.com/googleinterns/dagger-query/pull/36/commits/3cd296eac7b615908109897c271e1f064c673afc) ✅